trino/sqlalchemy/compiler.py

```diff
@@ -91,7 +91,16 @@
 
 
 class TrinoSQLCompiler(compiler.SQLCompiler):
-    pass
+    def limit_clause(self, select, **kw):
+        """
+        Trino support only OFFSET...LIMIT but not LIMIT...OFFSET syntax.
+        """
+        text = ""
+        if select._offset_clause is not None:
+            text += "\n OFFSET " + self.process(select._offset_clause, **kw)
```

Comment:
Remove space before `OFFSET`
```suggestion
            text += "\nOFFSET " + self.process(select._offset_clause, **kw)
```